src/main/java/at/hannibal2/skyhanni/config/features/foraging/ForagingConfig.kt

```diff
@@ -74,6 +75,14 @@ class ForagingConfig {
     @OnlyModern
     var mutePhantoms = true
 
+    @Expose
+    @ConfigOption(name = "Clean Tree View", desc = "Hides the floating blocks when mining trees in galatea.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    @OnlyModern
+    @SearchTag("fig mangrove")
+    var cleanTreeView = true
```

Comment:
Please move this is a new sub category "big trees" or something. There are other features planned for trees

Reply:
do you have a better name than "big trees"

Reply:
No not really

Reply:
I don't think this needs its own category - everything in here is Galatea specific already 🤷

Reply:
> I don't think this needs its own category - everything in here is Galatea specific already 🤷

skyhanni has more features in the galatea category except trees